model/src/main/java/com/basistech/rosette/dm/TagSet.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2019 Basis Technology Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.basistech.rosette.dm;
+
+/**
+ * Enumeration for part of speech tag sets used in Basis products.
+ */
+public enum TagSet {
+
+    /**
+     * UPT-16 version 1 tag set
+     */
+    UPT16_V1,
+
+    /**
+     * Basis Technology's language neutral tag set, used for emojis, emoticons, hashtags, etc.
+     */
+    BT_LANGUAGE_NEUTRAL,
+
+    /**
+     * Basis Technology's tag set for Arabic
+     */
+    BT_ARABIC,
+
+    /**
+     * Basis Technology's CLA tag set for Chinese
+     */
+    BT_CHINESE,
+
+    /**
+     * Basis Technology's non-CLA tag set for Chinese
+     */
+    BT_CHINESE_2,
+
+    /**
+     * Basis Technology's tag set for Czech
+     */
+    BT_CZECH,
+
+    /**
+     * Basis Technology's tag set for Dutch
+     */
+    BT_DUTCH,
+
+    /**
+     * Basis Technology's tag set for English
+     */
+    BT_ENGLISH,
+
+    /**
+     * Basis Technology's tag set for French
+     */
+    BT_FRENCH,
+
+    /**
+     * Basis Technology's tag set for German
+     */
+    BT_GERMAN,
+
+    /**
+     * Basis Technology's tag set for Greek
+     */
+    BT_GREEK,
+
+    /**
+     * Basis Technology's tag set for Hebrew
+     * todo: Maybe Technion's?
+     */
+    BT_HEBREW,
+
+    /**
+     * Basis Technology's tag set for Hungarian
+     */
+    BT_HUNGARIAN,
+
+    /**
+     * Basis Technology's tag set for Italian
+     */
+    BT_ITALIAN,
+
+    /**
+     * Basis Technology's JLA tag set for Japanese
+     */
+    BT_JAPANESE,
+
+    /**
+     * Basis Technology's non-JLA tag set for Japanese
```

Comment:
We want to avoid the terms “CLA” and “JLA”.

Reply:
What other feature can be used to distinguish the tag sets?

Reply:
I don’t know; I was hoping you wouldn’t ask. Maybe we could call it `BT_JAPANESE_RBLJE2` because it was added in RBL-JE 2.0.0.

Reply:
Japanese pos tag set is part JLASS, Basis hired a company to produce it.
https://confluence.basistech.net/pages/viewpage.action?pageId=5572343

CLA https://confluence.basistech.net/display/ENG/CLA+lexicon+change+to+Wenlin+and+City+University+of+HK